Docker/README.md

```diff
@@ -10,8 +10,8 @@ the components necessary to run the O3DE demo project simulator through the O3DE
 * Ubuntu 20.04 (Focal) or 22.04 (Jammy)
 * At least 60 GB of free disk space
 * Docker installed and configured
+  * **Note** It is recommended to have Docker installed correctly and in a secure manner so that the docker commands in this guide do not require elevated priviledges (sudo) in order to run them. See [Docker Engine post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) for more details.
```

Comment:
Does the user still have to be part of the docker group to run sudoless? Maybe we can reference these instructions of so: https://docs.docker.com/engine/install/linux-postinstall/

Reply:
There are instructions to set it up so that the user does not have to be in the docker group ( https://docs.docker.com/engine/security/rootless/ ). But those instructions are also mentioned in the link provided in this update, and the user reading these instructions can decide which approach to take